scripts/config/storybook/webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const HardSourceWebpackPlugin = require('hard-source-webpack-plugin');
+const IgnoreNotFoundExportWebpackPlugin = require('ignore-not-found-export-webpack-plugin');
+const path = require('path');
+
+module.exports = ({ config }) => {
```

Comment:
I know this is merged already but can we share any of this with the regular webpack config?